common/src/main/java/org/tron/core/Constant.java

```diff
@@ -101,8 +101,8 @@ public class Constant {
   public static final String NODE_MAX_CONNECTIONS_WITH_SAME_IP = "node.maxConnectionsWithSameIp";
   public static final String NODE_MIN_PARTICIPATION_RATE = "node.minParticipationRate";
   public static final String NODE_LISTEN_PORT = "node.listen.port";
-  public static final String NODE_DISCOVERY_PUBLIC_HOME_NODE = "node.discovery.public.home.node";
-  public static final String NODE_DISCOVERY_PING_TIMEOUT = "node.discovery.ping.timeout";
+  //public static final String NODE_DISCOVERY_PUBLIC_HOME_NODE = "node.discovery.public.home.node";
+  //public static final String NODE_DISCOVERY_PING_TIMEOUT = "node.discovery.ping.timeout";
 
   public static final String NODE_P2P_PING_INTERVAL = "node.p2p.pingInterval";
```

Comment:
Why not delete NODE_P2P_PING_INTERVAL?

Reply:
Yes, there is something wrong when split pr. I have deleted it.